03-intro-telemetry-data.qmd

```diff
@@ -246,18 +244,20 @@ Now we have clean data to work with we can get to the fun data exploration part!
 ```{r read in geopackage}
 
 #bou = read.csv("clean_data/caribou.csv")
```

Comment:
What do you think about using `readr::read_csv()` so we're working with tibbles? If for nothing else than the nicer print methods?

Reply:
Yep that makes sense, I will update the other instances

Reply:
I now realise why i didnt as it changes all the heading names... oh well i will update these

Reply:
Oh that's right. There's a `name_repair` argument in `read_csv` which you can use to disable this

Reply:
Why does read_csv change the column headers from . to - while the read_xlsx does not? Does this seem odd to you? or am I missing something?